client/lib/application/src/application-list/application-list.hbs

```diff
@@ -3,14 +3,48 @@
     <a href="/application/create" class="btn btn-default">
     <i class="fa fa-plus"></i> Create Application</a>
 </div>
-{{#if apps.length}}
-    <ul>
-        {{#each apps}}
+<div class="form-group">
+    <form class="form">
+        <label for="yourturn-search">Search:</label>
+        <div class="input-group">
+            <input
+                name="yourturn-search"
+                autofocus="autofocus"
+                value="{{term}}"
+                type="search"
+                aria-label="Enter your term"
+                data-action="search"
+                placeholder="Application" />
```

Comment:
Maybe use `Kio` or `Pier One` here? Placeholders should be examples of what you can enter, nobody is going to search for `Application` :blush: